src/index.ts

```diff
@@ -22,6 +22,7 @@ export * as IEventTarget from "@hyperion/hyperion-dom/src/IEventTarget";
 export * as IElement from "@hyperion/hyperion-dom/src/IElement";
 export * as IHTMLElement from "@hyperion/hyperion-dom/src/IHTMLElement";
 export * as ICSSStyleDeclaration from "@hyperion/hyperion-dom/src/ICSSStyleDeclaration";
+// export * as IGlobalEventHandlers from "@hyperion/hyperion-dom/src/IGlobalEventHandlers";
```

Comment:
I left this here as a reminder and for consistency with other bundler, but as of now we don't need to worry about exposing this to the outside world!